inspirehep/modules/search/views.py

```diff
@@ -69,6 +69,12 @@ def search():
                            **ctx)
 
 
+@blueprint.route("/ping")
+def ping():
+    """Ping for load balancer."""
+    return "OK", 200
```

Comment:
Uh? Who is looking for this guy under `/search/ping`? It should be pointing to `/ping`...